### PR TITLE
Update FTE docs to add HDFS as exchange type

### PR DESCRIPTION
**Pull request by sanchitkashyap**
## Description
Updating FTE docs to add HDFS as exchange manager type and for spooling. Added details about configs needed to configure HDFS exchange manager.

## Additional context and related issues
Refer [PR](https://github.com/trinodb/trino/pull/15160) for relevant code changes.

## Release notes

( x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

### CODE REVIEW

File: docs/src/main/sphinx/admin/fault-tolerant-execution.rst [hunk 1]
```diff
@@ -458,6 +459,15 @@ the property may be configured for:
        retry a request.
      - ``10``
      - Azure Blob Storage
+   * - ``exchange.hdfs.block-size``
+     - Block size for HDFS storage.
+     - ``4MB``
+     - HDFS
+   * - ``hdfs.config.resources``
+     - Comma-separated list of HDFS configuration files. These files must
```
Comment (mosabua) [inline docs/src/main/sphinx/admin/fault-tolerant-execution.rst:467]: its a list of paths to the conf files.. 

Comma-separated list of paths to HDFS configuration files, for example ``/etc/hdfs-site.xml`` The files must exist on all nodes in the Trino cluster.


Related question.. are relative paths allowed? If yes.. what are they relative to?

Comment (sanchitkashyap) [reply docs/src/main/sphinx/admin/fault-tolerant-execution.rst]: Updated the sentence.
Only absolute paths will work.

File: docs/src/main/sphinx/admin/fault-tolerant-execution.rst [hunk 2]
```diff
@@ -344,15 +344,16 @@ Exchange spooling is responsible for storing and managing spooled data for
...
+or :ref:`HDFS <fte-exchange-hdfs>`.
 
 Configuration
 ^^^^^^^^^^^^^
 
 To configure an exchange manager, create a new
 ``etc/exchange-manager.properties`` configuration file on the coordinator and
 all worker nodes. In this file, set the ``exchange-manager.name`` configuration
-propertry to ``filesystem``, and additional configuration properties as needed
+property to ``filesystem`` or ``hdfs``, and additional configuration properties as needed
```
Comment (colebow) [inline docs/src/main/sphinx/admin/fault-tolerant-execution.rst:356]:
```suggestion
property to ``filesystem`` or ``hdfs``, and set additional configuration properties as needed
```
nit, to make it grammatically correct & clear

File: docs/src/main/sphinx/admin/fault-tolerant-execution.rst [hunk 3]
```diff
@@ -536,6 +546,20 @@ GCS bucket as the spooling storage destination.
     exchange.s3.endpoint=https://storage.googleapis.com
     exchange.gcs.json-key-file-path=/path/to/gcs_keyfile.json
 
+.. _fte-exchange-hdfs:
+
+HDFS
+~~~~~~~~~~~~~~~~~~
+
+The following example ``exchange-manager.properties`` configuration specifies HDFS
```
Comment (colebow) [inline docs/src/main/sphinx/admin/fault-tolerant-execution.rst:554]:
```suggestion
The following ``exchange-manager.properties`` configuration example specifies HDFS
```
nit: clearer when re-ordered this way

**Review by colebow — APPROVED** [review]
Two nits but otherwise looks good.

File: docs/src/main/sphinx/admin/fault-tolerant-execution.rst (hunk 2 above)
Comment (sanchitkashyap) [reply docs/src/main/sphinx/admin/fault-tolerant-execution.rst]: Done

File: docs/src/main/sphinx/admin/fault-tolerant-execution.rst (hunk 3 above)
Comment (sanchitkashyap) [reply docs/src/main/sphinx/admin/fault-tolerant-execution.rst]: Done

File: docs/src/main/sphinx/admin/fault-tolerant-execution.rst [hunk 4]
```diff
@@ -536,6 +546,20 @@ GCS bucket as the spooling storage destination.
     exchange.s3.endpoint=https://storage.googleapis.com
     exchange.gcs.json-key-file-path=/path/to/gcs_keyfile.json
 
+.. _fte-exchange-hdfs:
+
+HDFS
+~~~~~~~~~~~~~~~~~~
```
Comment (mosabua) [inline docs/src/main/sphinx/admin/fault-tolerant-execution.rst:552]: Shorten this to be same length as title itself

**Review by mosabua — APPROVED** [review]
One mini update .. then good to go

File: docs/src/main/sphinx/admin/fault-tolerant-execution.rst (hunk 4 above)
Comment (sanchitkashyap) [reply docs/src/main/sphinx/admin/fault-tolerant-execution.rst]: Done

**Comment by mosabua** [comment]
Thanks @arhimondr !
